benchmark/scripts/run_smoke_bench

```diff
@@ -80,10 +83,6 @@ def main():
     argparser.add_argument(
         'newbuilddir', nargs=1, type=str,
         help='new benchmark build directory')
-    argparser.add_argument(
-        '-check-added', action='store_const',
-        help="Run BenchmarkDoctor's check on newly added benchmarks",
-        const=lambda args: check_added(args), dest='func')
     argparser.set_defaults(func=test_opt_levels)
```

Comment:
Can you please remove "func" and call test_opt_levels directly.
We don't need it anymore and it will simplify the code.

Reply:
Done.